src/app/components/AllowanceMenu/index.tsx

```diff
@@ -103,6 +105,9 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
               onChange={(e) => setBudget(e.target.value)}
             />
           </div>
+          <p className="my-1 dark:text-white">
+            This will reset the current budget
```

Comment:
@dylancom you think we should an optional *hint*-text to inputs?

Reply:
Nice idea. In italic / a lighter grey for e.g.?

Reply:
![image](https://user-images.githubusercontent.com/1016218/173497955-75d8de87-567a-46de-baf5-d3895e25ae46.png)